pynucastro/eos/fermi_integrals.py

```diff
@@ -20,6 +20,8 @@
 
 from .quadrature_weights import w_lag, w_leg, x_lag, x_leg
 
+MAX_EXPONENT = np.trunc(np.log(np.finfo(np.float64).max))
```

Comment:
This is an evil good way to get the maximum allowed exponent for the double precision type  `np.float64`, for the `np.exp()` function. I am glad this limit is found.